source/_patterns/01-molecules/navigation/view-selector.mustache

```diff
@@ -1,13 +1,13 @@
-<div class="view-selector{{#figureUrl}} view-selector--has-figures{{/figureUrl}}" data-behaviour="ViewSelector"{{#sideBySideUrl}} data-side-by-side-link="{{sideBySideUrl}}"{{/sideBySideUrl}}>
+<div class="view-selector{{#secondaryUrl}} view-selector--has-secondary{{/secondaryUrl}}" data-behaviour="ViewSelector"{{#sideBySideUrl}} data-side-by-side-link="{{sideBySideUrl}}"{{/sideBySideUrl}}>
   <ul class="view-selector__list">
-    <li class="view-selector__list-item view-selector__list-item--article{{^figureIsActive}} view-selector__list-item--active{{/figureIsActive}}">
-      <a href="{{articleUrl}}" class="view-selector__link view-selector__link--article"><span>Article</span></a>
+    <li class="view-selector__list-item view-selector__list-item--main{{^secondaryIsActive}} view-selector__list-item--active{{/secondaryIsActive}}">
+      <a href="{{mainUrl}}" class="view-selector__link view-selector__link--main"><span>Article</span></a>
```

Comment:
If we intend to use the view-selector template do we need to also add a parameter for `mainUrlLabel` and `secondaryUrlLabel`?

Reply:
Thanks, it is part of the [PR](https://github.com/elifesciences/pattern-library/pull/1178) for Tabs in Home page but it is better to be part of this one

Reply:
Done, hardcoded text is removed